src/index.js

```diff
@@ -181,9 +180,24 @@ function nativePlugin(options) {
                     renamedMap.set(nativePath, mapping);
                     isNew = true;
                 }
-
                 if (isNew) {
-                    Fs.copyFileSync(nativePath, mapping.copyTo);
+                    if (Fs.pathExistsSync(nativePath))
```

Comment:
So maybe here do a preliminary step - check for existence, and then go through the mapping function.  
If the function does not exist, the path does not change. If there is a mapper and the  result if `undefined` or `null` - also leave it unchanged.  

Finally, try to copy the file. Failure will emit a warning.

In the middle - if there's some complex logic to perform to provide the "replacement", the mapper function will do it. Including checking for existence of replacements etc.

This will space for almost any quirk anyone will want to use there.